### PR TITLE
Updated the Credentials Service to use Python 3.5

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-7552

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/devops @edx/ecommerce please review.

**Comment by feanil** [comment]
Looks like the docker build failed @clintonb

File: playbooks/roles/credentials/tasks/main.yml [hunk 1]
```diff
@@ -30,17 +30,42 @@
...
+    - install
+    - install:system-requirements
+
+- name: install nodenv
   pip:
-    requirements: "{{ credentials_requirements_base }}/{{ item }}"
-    # Ansible will initialize this virtualenv if it's missing.
-    virtualenv: "{{ credentials_venv_dir }}"
-    state: present
+    name: "nodeenv"
```
Comment (rlucioni) [inline playbooks/roles/credentials/tasks/main.yml:61]: Why do we need to mess with nodeenv as part of this?

**Comment by clintonb** [comment]
Updating the code to retain backward-compatibility with Python 2.7.

File: playbooks/roles/credentials/tasks/main.yml (hunk 1 above)
Comment (clintonb) [reply playbooks/roles/credentials/tasks/main.yml:61]: We use a separate nodeenv for sandboxes and Vagrant-based devstacks. I'm not sure how we managed to build AMIs without installing this package via `pip`. It's no longer in `production.txt`.

**Comment by clintonb** [comment]
Merge https://github.com/edx/credentials/pull/201 first

**Comment by clintonb** [comment]
@feanil @rlucioni thumbs?

**Review by rlucioni — APPROVED** [review]

**Pull request merged**